src/components/Icon/Icon.tsx

```diff
@@ -11,6 +11,7 @@ type Color =
   | 'critical'
   | 'warning'
   | 'highlight'
+  | 'interactive'
```

Comment:
I noticed that #4112 adds `interactive` to `Icon` as well – will rebase after that's merged. cc @kyledurand

Reply:
Oops totally forgot about that. I'll get it shipped asap